sw/device/silicon_creator/rom/data/rom_e2e_testplan.hjson

```diff
@@ -843,7 +843,11 @@
             '''
       tags: ["rom", "verilator", "dv", "fpga", "silicon"]
       stage: V2
-      tests: []
+      tests: [
+        "//sw/device/examples/sram_program:sram_program_fpga_cw310_test_otp_dev",
```

Comment:
Do we have an Hjson linter?

Reply:
@cfrantz @milesdai @timothytrippel @gregAC Do we?

Reply:
yeah I don't think so either.